src/peripherals/scales.cpp

```diff
@@ -19,10 +19,10 @@ void scalesInit(float scalesF1, float scalesF2) {
     LoadCells.set_scale(scalesF1, scalesF2);
     LoadCells.power_up();
 
-    delay(500);
+    //delay(500);
```

Comment:
Should we kill this line if it's no longer needed?

Reply:
yeah another omission